app/src/main/res/layout/item_coin_balance.xml

```diff
@@ -9,15 +9,28 @@
     android:paddingVertical="16dp">
 
     <ImageView
+        android:id="@+id/logo_image_background"
```

Comment:
can't we just set background for logo_image?

Reply:
That is exactly the problem with this ticket. When using background I found no way o get rid of the corona. Maybe you have an idea how to do this with a background?

Reply:
You can find an example with background  in the details screen.

Reply:
![image](https://user-images.githubusercontent.com/246473/104434093-1c630a80-558b-11eb-8426-f65f50378844.png)

Reply:
Have you tried: https://developer.android.com/reference/android/view/View#setClipToOutline(boolean) ?